packages/eslint-plugin/src/index.ts

```diff
@@ -44,6 +46,8 @@ const config = {
         '@elastic/eui/require-aria-label-for-modals': 'warn',
         '@elastic/eui/consistent-is-invalid-props': 'warn',
         '@elastic/eui/sr_output_disabled_tooltip': 'warn',
+        '@elastic/eui/no-css_color': 'warn',
+        '@elastic/eui/prefer_eui_icon_tip': 'warn',
```

Comment:
@alexwizp we have an inconsistency here. All ESLint rules here are kebab-case but `sr_output_disabled_tooltip` and now `prefer_eui_icon_tip`. Let's use kebab-case consistently.

```suggestion
        '@elastic/eui/prefer-eui-icon-tip': 'warn',
```